scripts/monorepo/src/getDependencies.spec.js

```diff
@@ -50,32 +50,27 @@ describe(`#getDependencies`, () => {
         Object {
           "dependencyType": "devDependencies",
           "isTopLevel": true,
-          "name": "eslint-plugin",
+          "name": "scripts-tasks",
```

Comment:
style: nx source analyze different ordering effect - the functionality is the same